src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,120 @@
 package org.example;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
 public class Main {
+
+    static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
```

Comment:
CLEAN-CODE: globalni promenne v ramci tridy Main, by mely byt private, tedy bude k nim pristup jen z Main tridy. Vice se o objektove orientovanem programovani budeme ucit v lednu.

```suggestion
    private static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
```